gemma-cli/pom.xml

```diff
@@ -77,10 +77,5 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-api</artifactId>
         </dependency>
-        <dependency>
-            <groupId>ubc.chibi.compass-fork</groupId>
-            <artifactId>compass-fork</artifactId>
-            <version>1.1.1</version>
-        </dependency>
```

Comment:
This should be merged directly in the development branch, it's redundant with gemma-core.